packages/template-retail-react-app/app/pages/checkout/partials/shipping-options.jsx

```diff
@@ -236,7 +236,6 @@ export default function ShippingOptions() {
                                     textDecoration="line-through"
                                     color="gray.600"
                                     marginLeft={1}
-                                    aria-hidden="true"
```

Comment:
This was implemented because of strikethrough pricing, where previously it was unclear which was the real price as the screen reader would read out both prices. I tested this PR and it seems like we reverted back where the screen reader reads out both prices

I'm ok with removing the `aria-hidden` but we should make it clear what the real price is, which the `shippingPriceLabel` does

Reply:
Instead of removing aria-hidden, could we specify a role so the issue is addressed?

Reply:
Yes, added aria-hidden back and `role` attribute